modules/custom/az_user_expire/az_user_expire.info.yml

```diff
@@ -0,0 +1,9 @@
+name: 'Quickstart User Expire'
+type: module
+description: 'Enables automatic expiration of user access.'
+core_version_requirement: ^9 || ^10
```

Comment:
Could this be set to `^10 || ^11`?

Reply:
Not yet, because this module depends on config_sync which is not yet D11 compatible.

Reply:
It's a grey area I suppose.